circuit_knitting/cutting/cutting_reconstruction.py

```diff
@@ -26,49 +26,70 @@
 
 
 def reconstruct_expectation_values(
-    quasi_dists: Sequence[Sequence[Sequence[tuple[QuasiDistribution, int]]]],
+    results: SamplerResult | dict[str | int, SamplerResult],
     coefficients: Sequence[tuple[float, WeightType]],
     observables: PauliList | dict[str | int, PauliList],
 ) -> list[float]:
     r"""
     Reconstruct an expectation value from the results of the sub-experiments.
 
     Args:
-        quasi_dists: A 3D sequence of length-2 tuples containing the quasi distributions and
-            QPD bit information from each sub-experiment. Its expected shape is
-            (num_unique_samples, num_partitions, num_commuting_observ_groups)
-        coefficients: A sequence of coefficients, such that each coefficient is associated
-            with one unique sample. The length of ``coefficients`` should equal
-            the length of ``quasi_dists``. Each coefficient is a tuple containing the numerical
-            value and the ``WeightType`` denoting how the value was generated.
+        results: The results from running the cutting subexperiments. If the cut circuit
+            was not partitioned between qubits and run separately, the input should be
+            a :class:`~qiskit.primitives.SamplerResult` instance or a dictionary mapping
+            a single partition to the results. If the circuit was partitioned and its
+            pieces run separately, the input should be a dictionary mapping partition labels
```

Comment:
```suggestion
            pieces ran separately, this argument should be a dictionary mapping partition labels
```

Reply:
> 

I think "the pieces were run separately" is correct. I know this isn't phrased exactly like that, but I believe run is the past participle and should be used here? I'm a little rusty with the grammar, so I could be off :)

Reply:
That sounds even better to me.  I agree with your reasoning.
```suggestion
            pieces were run separately, this argument should be a dictionary mapping partition labels
```